app/assets/javascripts/survey_trip_stops.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
```

Comment:
Let me look more into it, I've never used it but it looks promising from my initial peek at it